### PR TITLE
[onkyo] HT-RC440 added

### PR DESCRIPTION
**Pull request by markus7017**
Definitions for HT-RC440 added

Signed-off-by: Markus Michels <markus7017@gmail.com>

### CODE REVIEW

**Comment by lolodomo** [comment]
Please run mvn spotless:apply
And please do not update the German translation in that PR.

**Comment by jlaur** [comment]
> please do not update the German translation in that PR

For reference: https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations

**Comment by markus7017** [comment]
ok, will do

**Comment by markus7017** [comment]
@lolodomo done

File: bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoModel.java [hunk 1]
```diff
@@ -24,6 +24,7 @@ public enum OnkyoModel {
 
     // Please also add new supported models to README.md
```
Comment (mlobstein) [inline bundles/org.openhab.binding.onkyo/src/main/java/org/openhab/binding/onkyo/internal/OnkyoModel.java:25]: The model number should be added to the README.

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**